src/web/components/elements/ItemLinkBlockElement.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { textSans } from '@guardian/src-foundations/typography';
+
+type Props = {
+	html: string;
+};
+
+const itemLinkStyles = css`
+	${textSans.medium()}
+	font-weight: 400;
```

Comment:
You can use `fontWeight` in `textSans` here

Reply:
it was unusual that without `font-weight: 400;` the styles from global would take priority (although it is defined higher up in the DOM tree).